project-base/app/src/Controller/Admin/SideMenuConfigurationSubscriber.php

```diff
@@ -197,17 +197,17 @@ public function configureSettingsMenu(ConfigureMenuEvent $event): void
         $storeMenu->addChild('new_store', ['route' => 'admin_store_new', 'display' => false, 'label' => t('New store')]);
         $storeMenu->addChild('edit_store', ['route' => 'admin_store_edit', 'display' => false, 'label' => t('Edit store')]);
 
-        $parameterValueMenu = $listMenu->addChild('parameter_values', ['route' => 'admin_parametervalue_list', 'label' => t('Hodnota parametru typu barva')]);
-        $parameterValueMenu->addChild('parameter_values_edit', ['route' => 'admin_parametervalue_edit', 'display' => false, 'label' => t('Editace hodnoty parametru typu barva')]);
+        $parameterValueMenu = $listMenu->addChild('parameter_values', ['route' => 'admin_parametervalue_list', 'label' => t('Pararameter value of type color')]);
+        $parameterValueMenu->addChild('parameter_values_edit', ['route' => 'admin_parametervalue_edit', 'display' => false, 'label' => t('Editing parameter value of type color')]);
 
         $transportTypeMenu = $listMenu->addChild('transport_type', ['route' => 'admin_transporttype_list', 'label' => t('Transport types')]);
         $transportTypeMenu->addChild('transport_type_edit', ['route' => 'admin_transporttype_edit', 'display' => false, 'label' => t('Edit transport type')]);
 
-        $stockMenu = $settingsMenu->addChild('stocks', ['label' => t('Skladovost')]);
-        $stockMenu->addChild('stock', ['route' => 'admin_stock_list', 'label' => t('Sklady')]);
-        $stockMenu->addChild('new_stock', ['route' => 'admin_stock_new', 'display' => false, 'label' => t('Nový sklad')]);
-        $stockMenu->addChild('edit_stock', ['route' => 'admin_stock_edit', 'display' => false, 'label' => t('Detail skladu')]);
-        $stockMenu->addChild('stock_settings', ['route' => 'admin_stock_settings', 'label' => t('Nastavení skladů')]);
+        $stockMenu = $settingsMenu->addChild('stocks', ['label' => t('Stocking')]);
+        $stockMenu->addChild('stock', ['route' => 'admin_stock_list', 'label' => t('Warehouses')]);
```

Comment:
Well I understand, but I thought renaming Stock entity and its classes to Warehouse would be out of scope of this change and word "Stock" does not mean the room / building where the goods are stored but it is amount of the goods so I felt wrong to use Stock when it would be incorrect and English speaking people would not understand it.

Reply:
I understand, in that case, I believe we should at least unify texts in translation files. 
We still have 
- `Default stock cannot be removed` 
- `Do you really want to remove this stock? By deleting this stock you will remove all stock quantities from products and association to stores. This step is irreversible!`
- `Stock`
- `Stock <strong><a href=\"{{ url }}\">{{ name }}</a></strong> created`
- `Stock <strong><a href=\"{{ url }}\">{{ name }}</a></strong> modified`
- `Stock <strong>{{ name }}</strong> deleted`
- `Stock <strong>{{ name }}</strong> was set as default.`
- `Stocks`
- `Stocks - full`
- `Stocks - view`

Reply:
Changed